airflow/www/templates/airflow/dags.html

```diff
@@ -105,7 +105,7 @@ <h2>DAGs</h2>
                     </span>
 
                     <div style="float: right; max-width: 70%; text-align: right; line-height: 160%;">
-                    {% for tag in dag.tags %}
+                    {% for tag in dag.tags | sort(attribute='name') %}
```

Comment:
Can you please add a test for it in test/www/views.py ?

Reply:
Or maybe sorting it python side https://github.com/apache/airflow/blob/master/airflow/models/dag.py#L1481-L1530

?

Reply:
I do not fully understand why (and how) this should be tested.

Reply:
> Why ?
So that if someone removes the sorting code, it should fail the test so we know that it is broken

> How ?
https://github.com/apache/airflow/blob/master/tests/www/test_views.py#L441 file has got test to check if "certain" text exists or not. 

--

The other part what I commented was that maybe we sort the list before we pass it to the UI

I am happy to merge it as it is for now, but we should take care of it.